lighting_import_attachment_product/models/lighting_import_attachment_file.py

```diff
@@ -0,0 +1,13 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class LightingImportAttachmentFile(models.Model):
+    _inherit = "lighting.import.attachment.file"
+
+    product_lines = fields.One2many(
+        comodel_name="lighting.import.attachment.file.product",
+        inverse_name="file_id",
+    )
```

Comment:
Define `required`, `tracking`, `cascade`, etc... and sql constraints (if any) at the very beginning of coding, if you don't there's a very high chances you forget.